test/functional-portable/corerp/noncloud/resources/testdata/containers/corerp-resources-friendly-container-version-1.bicep

```diff
@@ -19,7 +19,7 @@ resource webapp 'Applications.Core/containers@2023-10-01-preview' = {
     container: {
       image: magpieimage
       env: {
-        DBCONNECTION: redis.connectionString()
+        DBCONNECTION: redis.listSecrets().connectionString
```

Comment:
Was this wrong before?

Reply:
Or is it a new feature of the provider?

Reply:
Or is it a new feature of the provider?

Reply:
Yes, its a new feature so we now make function calls using listSecrets

Reply:
Yes, we're intentionally simplifying the design of the provider.